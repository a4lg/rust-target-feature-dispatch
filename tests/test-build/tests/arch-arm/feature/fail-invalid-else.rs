pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    if class("arm") {
        if "sha2" {
            1
        } else {
            // ERROR: reserved for internals.
            @__tgtfeat_dispatch_no_fallback
        }
    } else {
        0
    }
};
fn main() {}
