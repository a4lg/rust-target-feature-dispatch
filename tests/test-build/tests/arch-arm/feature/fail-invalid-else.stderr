error: invalid feature-specific `else` clause
  --> tests/arch-arm/feature/fail-invalid-else.rs:1:24
   |
1  |   pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
   |  ________________________^
2  | |     if class("arm") {
3  | |         if "sha2" {
4  | |             1
...  |
12 | | };
   | |_^
   |
   = note: this error originates in the macro `$crate::target_feature_dispatch` which comes from the expansion of the macro `target_feature_dispatch::target_feature_dispatch` (in Nightly builds, run with -Z macro-backtrace for more info)
